src/abstractdataframe/io.jl

```diff
@@ -164,128 +167,140 @@ function html_escape(cell::AbstractString)
     return cell
 end
 
-function _show(io::IO, ::MIME"text/html", df::AbstractDataFrame;
-               summary::Bool=true, eltypes::Bool=true, rowid::Union{Int, Nothing}=nothing)
+function _show(io::IO,
+               ::MIME"text/html",
+               df::AbstractDataFrame;
+               summary::Bool=true,
+               eltypes::Bool=true,
+               rowid::Union{Int, Nothing}=nothing,
+               title::String="",
```

Comment:
```suggestion
               title::AbstractString="",
```

Reply:
I have fixed this and pushed the commit.

Reply:
Thanks!